docs/manual/external-tools.tex

```diff
@@ -127,12 +137,69 @@
             compile.options.compilerArgs += [
                     "-Xbootclasspath/p:${configurations.checkerFrameworkAnnotatedJDK.asPath}"
             ]
+            options.fork = true
+            options.forkOptions.jvmArgs += ["-Xbootclasspath/p:${configurations.errorproneJavac.asPath}"]
         }
     }
 }
 \end{Verbatim}
 \end{mysmall}
 
+\item To run the checkers, build using the \<checkTypes> variant:
+\begin{Verbatim}
+gradlew assembleCheckTypes
+\end{Verbatim}
+\end{enumerate}
+
+\subsectionAndLabel{JDK 11}{android-jdk11}
+This section shows what changes to make if you are using JDK 11.  They should work for JDK 9+, but
+we have only tested them with JDK 11.
+
+\begin{enumerate}
+
+\item In your module's \<build.gradle> file, set the source and target
+  compatibility to \<JavaVersion.VERSION\_1\_8>:
+
+\begin{Verbatim}
+android {
+    ...
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+\end{Verbatim}
+
+\item Add a build variant for running checkers:
+
+ \begin{Verbatim}
+ android {
+    ...
+      buildTypes {
+      ...
+        checkTypes {
+            javaCompileOptions.annotationProcessorOptions.
+                    classNames.add("org.checkerframework.checker.nullness.NullnessChecker")
+            // You can pass options like so:
+            // javaCompileOptions.annotationProcessorOptions.arguments.put("warns", "")
+        }
+    }
+}
+\end{Verbatim}
+
+\item Declare the Checker Framework dependencies:
+
+\begin{mysmall}
+\begin{Verbatim}
+dependencies {
+    ... existing dependencies...
+    ext.checkerFrameworkVersion = '2.11.1'
+    implementation "org.checkerframework:checker-qual-android:${checkerFrameworkVersion}"
```

Comment:
If a user doesn't use any checker annotations, then this line could be `compileOnly`.

I'm building an Android library, and I'm using checker to ensure null-safety. However, since I'm only using this at compile-time, and I'm not including any checker annotations in my source, I can use `compileOnly`, and my users won't have to include checker in their classpath when running with my framework.